tools/azsdk-cli/Azure.Sdk.Tools.Cli.Tests/Tools/CheckAllToolTests.cs

```diff
@@ -208,5 +208,30 @@ public async Task RunPackageCheck_EnumValues_WorksCorrectly()
             Assert.IsTrue(spellingResult.ExitCode >= 0);
             Assert.IsTrue(snippetsResult.ExitCode >= 0);
         }
+
+        [Test]
+        public async Task RunPackageCheck_WithCspellFixEnabled_WhenFileWithTypos_ReturnsResult()
+        {
+            // Arrange - Create a file with obvious spelling errors
+            var testFile = Path.Combine(_testProjectPath, "test_fix.md");
+            await File.WriteAllTextAsync(testFile, "This file contians obvioius speling erors.");
+
+            // Act - Test both regular cspell check and with fix enabled
+            var normalResult = await _packageCheckTool.RunPackageCheck(_testProjectPath, PackageCheckType.Cspell, false);
+            var fixResult = await _packageCheckTool.RunPackageCheck(_testProjectPath, PackageCheckType.Cspell, true);
+
+            // Assert
+            Assert.IsNotNull(normalResult);
+            Assert.IsNotNull(fixResult);
+            Assert.IsNotNull(normalResult.CheckStatusDetails);
+            Assert.IsNotNull(fixResult.CheckStatusDetails);
+            
+            // Both may succeed or partially succeed depending on environment and cspell version
+            Assert.That(normalResult.ExitCode, Is.GreaterThanOrEqualTo(0));
+            Assert.That(fixResult.ExitCode, Is.GreaterThanOrEqualTo(0));
+            
+            // The fix result should have different next steps indicating LLM guidance
+            Assert.IsTrue(fixResult.NextSteps != null && fixResult.NextSteps.Any());
```

Comment:
I would to `IsNotEmpty` here, otherwise if it fails the test output will just say "expected 'false' to be true" with no indication of what the assertion was aside from a line number.